### DISCUSSION TITLE
pixel vs. voxel scaling

### DISCUSSION THREAD

**Discussion by Curf** [PMEAL/porespy#798] [Q&A]
2. In snow algorithm by default voxel_size = 1, so can we think it is 1um/pixel of generated image?

Hello,

I have the same question regarding the conversion of pixels to voxels or vice versa - I had assumed it was the treated the same.  I am working with SEM images of pores of known size and um/pixel ratio (~0.013 um/pixel) and the distribution values seem to be off by a factor of 10 - is this due to some conversion within the code somewhere?

**Comment by jgostick** [comment]
The voxel size doesn't really have units, but I suggest using SI, so m/vx.  If you have 0.013 um/voxels, then you have 13e-9.  The the results you get back are in SI too.
